ctapipe/io/eventsource.py

```diff
@@ -198,3 +226,10 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
+
+
+def from_config(config, *args, **kwargs):
```

Comment:
Should restore tool argument, explicit arguments are better to have

Reply:
`tool` was not used in the body and was only forwarded to the client here. If we ever decide to get rid of `tool` or to rename `tool` this method does not need to change.